### ISSUE TITLE
[EN] [Event] [Hallow'sEnd] [NPC] [Quest] [Alliance] Jesper, Hallow's End Treats for Jesper!

### ISSUE THREAD

**Issue by Thefakemccoy** [Atlantiss/BugTracker#4747]
Bildo not sire if this was missed or you havent gotten to it.

First Jesper is not outside with the rest. He use to be in the orphange but was moved out in 2011 with the new updates.
Jesper:http://www.wowhead.com/npc=15310/jesper

He should offer the quest: http://www.wowhead.com/quest=8311/hallows-end-treats-for-jesper

Which should then allow you to do the following quest; 
(These quest should not be obtainable until you accept the main quest from jasper)
Stormwind
http://www.wowhead.com/quest=8356/flexing-for-nougat
Ironforge
http://www.wowhead.com/quest=8355/incoming-gumdrop
http://www.wowhead.com/quest=8353/chicken-clucking-for-a-mint
Darnassus
http://www.wowhead.com/quest=8357/dancing-for-marzipan

**Comment by Noviush** [comment]
Quest basicly works fine, except of Gnome. A few players confirmed on world chat, that innkeeper Firebrew in IF does not gives subquest for chicken emote. Other races have no problem with that.

**Comment by Aival94** [comment]
Quest is being offered normally but there are some issues:
All 4 quests [Chicken Clucking for a Mint](http://www.wowhead.com/quest=8353/chicken-clucking-for-a-mint), [Dancing for Marzipan](http://www.wowhead.com/quest=8357/dancing-for-marzipan), [Flexing for Nougat](http://www.wowhead.com/quest=8356/flexing-for-nougat) and [Incoming Gumdrop](http://www.wowhead.com/quest=8355/incoming-gumdrop) can be taken without having in log the quest [Hallow's End Treats for Jesper!](http://www.wowhead.com/quest=8311/hallows-end-treats-for-jesper)

[Chicken Clucking for a Mint](http://www.wowhead.com/quest=8353/chicken-clucking-for-a-mint) does not need for you to target the npc and /chicken on him but only need to speak on him and quest get autocompleted.
[Dancing for Marzipan](http://www.wowhead.com/quest=8357/dancing-for-marzipan) does not need for you to target the npc and /dance on her but only need to speak on her and quest get autocompleted.
[Flexing for Nougat](http://www.wowhead.com/quest=8356/flexing-for-nougat) does not need for you to target the npc and /flex on her but only need to speak on her and quest get autocompleted.

**Comment by aendry** [comment]
the clucking for a mint is bugged can't complete it

**Comment by Aival94** [comment]
@aendry Alliance or Horde. I did on Alliance just yesterday by just speaking to npc a second time.

**Comment by aendry** [comment]
I tried several times and nothing happened @aival94

**Comment by Aival94** [comment]
Ok. To be honest did not tried to press the complete button. Yeah After you complete quest  the complete button stay gray for some reason.

**Comment by SlyFoxUK** [comment]
I'm annoyed about this because i did all the Treat buckets on my Hunter, had 250 treats and threw them away as i didn't realise you could buy stuff with them, had a GM come to speak to me - "We can't give you those items back even though we can see you've done EVERY single bucket in the game in one week, we can't help you, you'll have to do the dailys and farm the Treats back as you have enough time to do that"... That'd be great if the dailys worked, this one gives 10!!!!!! I'm not going to be able to get anything unless this is fixed! And even Blizzard have a service for restoring DELETED items once per month!! IF Atlantis thinks i will ever pay any real money for here after this they are dead wrong! The Gm basically came and dangled the Magic Broom in my face by going "So you want this "Insert Magic Broom Link" , "you deffo sure you want the Magic Broom?" Making it sound like he was going to just give me one as i had proof i'd made a mistake and i had actually collected so many. Then he was like well screw you pal, go farm some more, we're not helping you for making a mistake!

**Comment by Samineh** [comment]
That quest is not a daily so it will give you 10 treats once only. You didn't even search in bugtracker before posting a new issue. Since you didn't get the result you wanted from the GM you are whining here which won't solve anything. About the daily quests only one of them may not work, however that one is tricky also. You just haven't tried to find a solution for that. About the daily reward and what the GM said, he is right: you can get the broom, the pet and the helm from the daily reward which saves you 450 treats.

**Comment by aendry** [comment]
Yep, I'm discarded the broom so much time that I can have a broom shop easily xD I suppose the helm can be used for transmog. The pet is easy too... Then you have the mount that's other story

**Comment by SlyFoxUK** [comment]
Actually Sam it says on WoWhead that it was repeatable.

**Comment by Samineh** [comment]
http://www.wowhead.com/quest=8311/hallows-end-treats-for-jesper
Can you show me where it does say about the quest being repeatable? The solution to the stink bomb daily quest is to try doing it until you can get it to work (one day I retried for 20 times till I made it). If I am pissed I will nag to my friends xD. I am just saying these because it will be unfair for the GMs and Devs who actually try to help and fix things and I am a player who gets frustrated just like you in times and you won't see me nagging on forum or bugtracker ;).
